image_recognition_util/src/image_recognition_util/image_reader.py

```diff
@@ -3,7 +3,7 @@
 import fnmatch
 
 
-def read_annotated(dir_path, patterns=["*.jpg", "*.png", "*.jpeg"]):
+def read_annotated(dir_path, patterns=["*.jpg", "*.png", "*.jpeg"], include_path=False):
```

Comment:
is include_path=False ever used? changing the return type base on input argument is kinda confusing.

Reply:
So the code would crash if input arguments are changing :p

Reply:
I know, the argument is there only for when you're testing. I found out that the reader kept repeating the same label over and over, also for paths of a different label. 

I'll remove the flag later on in this WIP.